src/modules/Builder/features/ReportConfigSetting/ReportConfigSetting.tsx

```diff
@@ -382,10 +382,19 @@ export const ReportConfigSetting = ({
   };
 
   useEffect(() => {
```

Comment:
I guess we don't need local state `successBannerVisible/setSuccessBannerVisible` anymore, since you could retrieve this info from `banners` store (via check if `SaveSuccessBanner` is in the list). Moreover, if you move `dispatch(banners.actions.addBanner({key: 'SaveSuccessBanner', ...` into a callback you could get rid of such places where used the code
```
 () => {
      setSuccessBannerVisible(true);
    },
```
and in that time no need `setSuccessBannerVisible(false);`